### ISSUE TITLE
Not working with Alex Mobs (beacon POI error)

### ISSUE THREAD

**Issue by Slither88** [Ephys/minecraft-magic-feather#17]
I am using version 4.00 of the mod with Alex's mobs version 1.18.6 and the game does not want to load as it says both magic feather and alex mobs is using the same beacon blockstate as a point of interest and its not allowed.

**Comment by ephys** [comment]
Thanks for letting me know! Let's hope there is a way to reuse POIs

**Comment by DarkYuan** [comment]
Yeah, that's a weird limitation. I assume it has to do with the Sunbird circling beacons mechanic. What does a POI do though?

Here's a crash log btw: https://gist.github.com/DarkYuan/0c4235ae49ec26b8f670bbe71c4a03f3

**Comment by ephys** [comment]
[I've just pushed an update that fixes it](https://www.curseforge.com/minecraft/mc-mods/magic-feather/files/3885050). I really hate the fix due to how brittle it is (I use Alex's mobs POI instead of my own if the mod is loaded) but at least it works

POI (points of interest) is an API to efficiently find blocks. We mark a block as a POI & minecraft will store these blocks in a list per chunk. Then we iterate that list to find the block in question.
They were made to allow villagers to find their work stations & beds, but are also used in vanilla to find nether portals, bee nests, lodestones (for the compass), & lightning rods.

Before POIs I iterated through the loaded tile entity list, which can be massive on modded, so I'm hoping using POIs is going to be better 🤞. If this breaks again I'll just revert back to iterating the tile entity list :/
